python/dgl/transform.py

```diff
@@ -4,15 +4,113 @@
 from scipy import sparse
 from ._ffi.function import _init_api
 from .graph import DGLGraph
+from . import backend as F
 from .graph_index import from_coo
 from .batched_graph import BatchedDGLGraph, unbatch
-from .backend import asnumpy, tensor
 
 
 __all__ = ['line_graph', 'khop_adj', 'khop_graph', 'reverse', 'to_simple_graph', 'to_bidirected',
-           'laplacian_lambda_max']
+           'laplacian_lambda_max', 'nearest_neighbor_graph', 'segmented_nearest_neighbor_graph']
```

Comment:
Please also mention these two functions in https://github.com/dmlc/dgl/blob/master/docs/source/api/python/transform.rst